### PR TITLE
update scope of who uses the tool

### PR DESCRIPTION
**Pull request by attritionorg**

### CODE REVIEW

**Review by yoder303 — APPROVED** [review]
LGTM

**Review by sickcodes — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
...
+C3 (Custom Command and Control) is a tool that allows Red Teams and [Malicious Ransomware Teams](https://www.fireeye.com/blog/threat-research/2021/05/shining-a-light-on-darkside-ransomware-operations.html) to rapidly develop and utilise esoteric command and control channels (C2). It's a framework that extends other red team tooling, such as the commercial Cobalt Strike (CS) product via [ExternalC2](https://www.cobaltstrike.com/downloads/externalc2spec.pdf), which is supported at release. It allows the Red Team to concern themselves only with the C2 they want to implement; relying on the robustness of C3 and the CS tooling to take care of the rest. This efficiency and reliability enable Red Teams to operate safely in critical client environments (by assuring a professional level of stability and security); whilst allowing for safe experimentation and rapid deployment of customised Tactics, Techniques and Procedures (TTPs). Thus, empowering Red Teams to emulate and simulate an adaptive real-world attacker.
```
Comment (Meatballs1) [inline README.md:9]: LMFTFY:

and Blue Teams to analyze and signature C2 mechanisms.

**Comment by attritionorg** [comment]
#SadPanda

**Pull request closed without merging**